notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -11793,6 +11793,54 @@ Christianity [The facility cook Sherman Dudley quoted several biblical passages,
 speculative genetic engineering [Dr. Susan McCallister genetically engineered some mako sharks to have bigger brains, making them smarter and more deadly as an unintended consequence.]
 
 
+movie: The Iron Giant (1999)
+============================
+
+:: Title
+The Iron Giant
+
+:: Description
+Set during the Cold War in 1957, the film centers on a young boy named Hogarth
+Hughes, who discovers and befriends a gigantic metallic robot who fell from
+outer space. With the help of a beatnik artist named Dean McCoppin, Hogarth
+attempts to prevent the U.S. military and Kent Mansley, a paranoid federal
+agent, from finding and destroying the Giant. It is based on the 1968 novel
+"The Iron Man" by Ted Hughes (which was published in the United States as
+"The Iron Giant") and was scripted by Tim McCanlies from a story treatment by
+Bird.
+
+:: Date
+1999-07-31
+
+:: References
+https://en.wikipedia.org/wiki/The_Iron_Giant
+
+:: Ratings
+4 <paul>
+
+:: Major Themes
+friendship [The socially isolated, nine-year-old boy Hogarth befriended the titular 50-foot tall metallic robot that'd fallen to Earth from outer space.]
+the Cold War [The film is set during the Cold War shortly after the Soviet Union shocked the world by launching the Sputnik 1 satellite into orbit. It explores how people coped with the tensions that existed between the Soviet Union and the United States from the point of view of both small town people and the U.S. military.]
+life in post-war America [The film offers a view of what life might have been like for people living in a small American town in the 1950s at a time when Cold War tensions were running high.]
+mother and son [Central to the story are the nine-year-old boy Hogarth and his widowed mother, Annie.]
+mechanical man [A main novelty of the film is the titular 50-foot tall, humanoid patterned robot that crashed from outer space off the coast of Maine.]
+giant monster [The gentle giant robot from outer space was taken to be a threat by the military and they launched a full-scale attack on it. The townspeople commonly referred to the giant robot as a monster.]
```

Comment:
This usage is noteworthy. Robots can be monsters too. I point out that the usage of a word like "monster" in the film is not itself evidence as words can subtly change meaning over time.